packages/account/src/Components/api-token/api-token.tsx

```diff
@@ -31,19 +29,12 @@ type AptTokenState = {
 };
 
 export type TApiToken = {
-    footer_ref: Element | DocumentFragment | undefined;
     is_app_settings: boolean;
-    overlay_ref:
-        | undefined
-        | ((...args: unknown[]) => unknown)
-        | import('prop-types').InferProps<{
-              current: import('prop-types').Requireable<unknown>;
-          }>;
     setIsOverlayShown: (is_overlay_shown: boolean | undefined) => void;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
 };
 
-const ApiToken = ({ footer_ref, is_app_settings, overlay_ref, setIsOverlayShown }: TApiToken) => {
+const ApiToken = ({ is_app_settings, setIsOverlayShown }: TApiToken) => {
```

Comment:
I think we can remove `is_app_settings`, its redundant and I could not see any place we are passing it in the component. 🙏

Reply:
Hi @utkarsha-deriv 👋 
Thanks for highlighting this. 

I checked that whole file is not being used so I have removed the file.